CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## 0.14.0 (March 30, 2023)
+
+### Enhancements
+
+ * Support for uri/uriabs Qute resolvers from Quarkus Renarde. See [#571](https://github.com/redhat-developer/quarkus-ls/issues/571).
+ * Workspace symbols and CodeLens for Renarde application REST endpoints. See [#777](https://github.com/redhat-developer/quarkus-ls/issues/777).
+ * CodeAction to insert expected `<input>` for Renarde `#form` section in Qute template. See [#778](https://github.com/redhat-developer/quarkus-ls/issues/778).
+ * Show validation errors for all Qute templates in a project, even unopened ones, without first needing to open a template. See [#813](https://github.com/redhat-developer/quarkus-ls/issues/813).
+ * Improve user tag snippet completion in Qute templates. See [#784](https://github.com/redhat-developer/quarkus-ls/issues/784).
+ * Validate user tag parameters in Qute templates. See [#788](https://github.com/redhat-developer/quarkus-ls/issues/788).
+ * Go to definition for user tag parameters in Qute templates. See [#789](https://github.com/redhat-developer/quarkus-ls/issues/789).

```

Comment:
I don't think combining all three of these is helpful, since it obfuscates what specifically was improved. i.e. a user of qute-ls can't read the changelog and try out the features without first having to read through the linked issues. I'll combine the last two, since they are related.

Reply:
Come to think of it "improve" isn't a great description of what was actually done, I'll reword that too.